account_invoice_partner_no_autofollow/models/res_config.py

```diff
@@ -0,0 +1,9 @@
+from odoo import fields, models
```

Comment:
rename file to res_config_settings.py
Check the OCA guidelines [here](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#file-naming)

Reply:
done